<launch>
  <arg name="fps" default="2" />              <!-- Possible values of fps are 1, 2, 4, 8, 16, 32, 64 -->
  <arg name="upscale_factor" default="20" />  <!-- Upscale factor with which to bicubically scale the
  image -->

  <node name="mlx90640" pkg="mlx90640_thermal_camera" type="mlx90640" output="screen">
    <param name="fps" value="$(arg fps)" />
    <param name="upscale_factor" value="$(arg upscale_factor)" />
  </node>
</launch>